modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2665,6 +2665,7 @@ by-x=By {0}
 by-x-on-x=By {0}, on {1}
 by-x-x=By <a href="{0}">{1}</a>, {2}
 bytes=Bytes
+c-product-id=C Product ID
```

Comment:
This is not something we can show to the citizen dev, please change it with something readable.